### ISSUE TITLE
Enhancement: ability to place blocks without block update

### ISSUE THREAD

**Issue by mojang2555** [SkriptLang/Skript#820]
set block at player to portal
set block above player to portal

they would break each other
its like with WorldEdit. but in worldedit there is //fast which would fix this.

My quetion is.. how to fix this

**Comment by Pikachu920** [comment]
This isn't really an *issue* but it'd be a good enchancement.

**Comment by TheBentoBox** [comment]
Yeah, it's definitely not a bug, it's the vanilla behavior. However, some sort of vanilla syntax similar to SkQuery's `@noupdate::` that lets you set a block without causing a block update would be nice and would likely resolve this issue.

**Comment by mojang2555** [comment]
So its possible with SkQuery?
im not good at script

so  how do i add it to my script

**Comment by mojang2555** [comment]
TheBentoBox

**Comment by mojang2555** [comment]
what

**Comment by mojang2555** [comment]
noupdate?

**Comment by bensku** [comment]
Do not spam, please.
